packages/core/src/interactions/packet/acknowledgement.ts

```diff
@@ -137,14 +134,20 @@ class PacketAcknowledgementInteraction<Chain extends HoprCoreConnector>
           }
         )
 
-        if (!(await this.node.paymentChannels.account.reservePreImageIfIsWinning(acknowledgedTicket))) {
```

Comment:
I think it's clearer before.

Reply:
Don't think so. Logical operations and Promises have been very often the source of runtime errors.

Reply:
I'm very unconvinced on this.

Reply:
Me too. But it doesn't harm us if we have it as proposed. Should (!) anyway be the same functionality.